include/circt/Dialect/RTG/IR/RTGOps.td

```diff
@@ -691,14 +691,12 @@ def VirtualRegisterOp : RTGOp<"virtual_reg", [
 
 //===- Test Specification Operations --------------------------------------===//
 
-def TestOp : RTGOp<"test", [
-  IsolatedFromAbove,
-  Symbol,
-  SingleBlock,
-  NoTerminator,
-  DeclareOpInterfaceMethods<OpAsmOpInterface, ["getAsmBlockArgumentNames"]>,
-  HasParent<"mlir::ModuleOp">
-]> {
+def TestOp
+    : RTGOp<"test", [IsolatedFromAbove, Symbol, SingleBlock, NoTerminator,
+                     DeclareOpInterfaceMethods<
+                         OpAsmOpInterface, ["getAsmBlockArgumentNames"]>,
+                     DeclareOpInterfaceMethods<SymbolUserOpInterface>,
+                     HasParent<"mlir::ModuleOp">]> {
```

Comment:
Super-nit: the rustfmt-style original formatting feels a bit more readable and less noisy in diffs than the clang-format-style formatting of the new code :grin: :bike: :house: